### PR TITLE
Tmz bugfix wig issue 1708

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 1]
```diff
@@ -1858,19 +1848,19 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
         LUAFile = LUAFile.replaceAll(key, 'objVariable_' + key);
       });
 
-      RegExp(r'deObfuscate\(".*?"\)').allMatches(LUAFile).forEach((obfuscatedText) {
-        var group = obfuscatedText.group(0);
-        if (group == null) return;
-        LUAFile = LUAFile.replaceAll(group, _deObfuscate(group));
-      });
+      // RegExp(r'deObfuscate\(".*?"\)').allMatches(LUAFile).forEach((obfuscatedText) {
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1851]: clean code

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart [hunk 2]
```diff
@@ -1830,7 +1820,7 @@ class _WherigoAnalyzeState extends State<WherigoAnalyze> {
 
   String _normalizeLUA(String LUAFile, bool deObfuscate) {
     if (deObfuscate) {
-      LUAFile = LUAFile.replaceAll(WherigoCartridgeLUAData.ObfuscatorFunction, 'deObfuscate');
+      //LUAFile = LUAFile.replaceAll(WherigoCartridgeLUAData.ObfuscatorFunction, 'deObfuscate');
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_analyze.dart:1823]: clean code

File: lib/tools/wherigo/wherigo_analyze/logic/wherigo_analyze_lua_messages.dart [hunk 3]
```diff
@@ -140,9 +140,10 @@ List<List<WherigoActionMessageElementData>> _getAllMessagesAndDialogsFromLUA(
                 _singleMessageDialog.add(WherigoActionMessageElementData(
                     ActionMessageType: WHERIGO_ACTIONMESSAGETYPE.IMAGE, ActionMessageContent: line));
               }
-            } else if (lines[i].trimLeft().startsWith('Text = ') ||
-                lines[i].trimLeft().startsWith('Text = ' + _obfuscatorFunction + '(') ||
-                lines[i].trimLeft().startsWith('Text = (' + _obfuscatorFunction + '(')) {
+            } else if (lines[i].trimLeft().startsWith('Text = ') //||
+            //    lines[i].trimLeft().startsWith('Text = ' + _obfuscatorFunction + '(') ||
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/logic/wherigo_analyze_lua_messages.dart:144]: clean code

File: lib/tools/wherigo/wherigo_analyze/logic/wherigo_analyze_lua_answers.dart [hunk 4]
```diff
@@ -58,21 +58,23 @@ WherigoAnswer _analyzeAndExtractOnGetInputSectionData(List<String> onGetInputLin
       do {
         i++;
         onGetInputLines[i] = onGetInputLines[i].trim();
-        if (!(onGetInputLines[i].trim() == '}' || onGetInputLines[i].trim() == '},')) {
-          if (onGetInputLines[i].trimLeft().startsWith(_obfuscatorFunction)) {
+        if (!(onGetInputLines[i] == '}' || onGetInputLines[i] == '},')) {
+          int obfuscation = _getObfuscatorFunction(onGetInputLines[i], _obfuscatorFunction);
+          //if (onGetInputLines[i].startsWith()) {
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/logic/wherigo_analyze_lua_answers.dart:63]: clean code

File: lib/tools/wherigo/wherigo_analyze/widget/wherigo_widget_output_answers.dart [hunk 5]
```diff
@@ -36,31 +37,41 @@ List<List<String>> _buildOutputListAnswers(BuildContext context, WherigoInputDat
...
-    String _variable = answers.length > 1 ? _answerIsVariable(answers[1]) : '';
+     String _variable = answers.length > 1 ? _answerIsVariable(answers[1]) : '';
+
     if (_variable.isNotEmpty) {
-      result = [
-        [i18n(context, 'wherigo_output_answer'), _variable, '']
-      ];
+      result.add([i18n(context, 'wherigo_output_answer'), _variable, '']);
+    }
+    //else {
```
Comment (Mike-3) [inline lib/tools/wherigo/wherigo_analyze/widget/wherigo_widget_output_answers.dart:45]: clean code

**Review by Mike-3 — CHANGES_REQUESTED** [review]

**Comment by t-m-z** [comment]
Done

**Pull request merged**